diff -ur ./fpdf/fpdf.php /tmp/vendor/setasign/fpdf/fpdf.php
--- ./fpdf/fpdf.php	2023-12-20 11:28:55.188649304 +0100
+++ /tmp/vendor/setasign/fpdf/fpdf.php	2023-12-20 11:28:00.669771775 +0100
@@ -1,18 +1,20 @@
 <?php
 /*******************************************************************************
 * FPDF                                                                         *
 *                                                                              *
 * Version: 1.86                                                                *
 * Date:    2023-06-25                                                          *
 * Author:  Olivier PLATHEY                                                     *
 *******************************************************************************/
+//erics
+namespace setasign\Fpdi;

 class FPDF
 {
 const VERSION = '1.86';
 protected $page;               // current page number
 protected $n;                  // current object number
 protected $offsets;            // array of object offsets
 protected $buffer;             // buffer holding in-memory PDF
 protected $pages;              // array containing pages
 protected $state;              // current document state

--- ./fpdi/src/FpdfTpl.php	2023-12-20 20:38:41.727085537 +0100
+++ /tmp/vendor/setasign/fpdi/src/FpdfTpl.php	2023-12-20 20:39:35.190539185 +0100
@@ -8,14 +8,14 @@
  * @license   http://opensource.org/licenses/mit-license The MIT License
  */

 namespace setasign\Fpdi;

 /**
  * Class FpdfTpl
  *
  * This class adds a templating feature to FPDF.
  */
-class FpdfTpl extends \FPDF
+class FpdfTpl extends \setasign\Fpdi\FPDF
 {
     use FpdfTplTrait;
 }
