diff -ur vendor.orig/setasign/fpdi/src/FpdiTrait.php vendor/setasign/fpdi/src/FpdiTrait.php
--- vendor.orig/setasign/fpdi/src/FpdiTrait.php	2024-05-15 14:09:48.422793420 +0200
+++ vendor/setasign/fpdi/src/FpdiTrait.php	2024-05-15 14:08:03.051568467 +0200
@@ -33,10 +33,10 @@
 use setasign\Fpdi\PdfReader\PageBoundaries;
 use setasign\Fpdi\PdfReader\PdfReader;
 use setasign\Fpdi\PdfReader\PdfReaderException;
-use /* This namespace/class is used by the commercial FPDI PDF-Parser add-on. */
-    /** @noinspection PhpUndefinedClassInspection */
-    /** @noinspection PhpUndefinedNamespaceInspection */
-    setasign\FpdiPdfParser\PdfParser\PdfParser as FpdiPdfParser;
+// use /* This namespace/class is used by the commercial FPDI PDF-Parser add-on. */
+//     /** @noinspection PhpUndefinedClassInspection */
+//     /** @noinspection PhpUndefinedNamespaceInspection */
+//     setasign\FpdiPdfParser\PdfParser\PdfParser as FpdiPdfParser;
 
 /**
  * The FpdiTrait
@@ -133,10 +133,10 @@
         // psr-4 (https://www.php-fig.org/psr/psr-4/) says: Autoloader implementations MUST NOT throw
         // exceptions, MUST NOT raise errors of any level, and SHOULD NOT return a value.
         /** @noinspection PhpUndefinedClassInspection */
-        if (\class_exists(FpdiPdfParser::class)) {
-            /** @noinspection PhpUndefinedClassInspection */
-            return new FpdiPdfParser($streamReader, $parserParams);
-        }
+        // if (\class_exists(FpdiPdfParser::class)) {
+        //     /** @noinspection PhpUndefinedClassInspection */
+        //     return new FpdiPdfParser($streamReader, $parserParams);
+        // }
 
         return new PdfParser($streamReader);
     }
