diff -ur vendor.orig/horstoeko/zugferd/src/ZugferdDocumentPdfBuilderAbstract.php vendor/horstoeko/zugferd/src/ZugferdDocumentPdfBuilderAbstract.php
--- vendor.orig/horstoeko/zugferd/src/ZugferdDocumentPdfBuilderAbstract.php	2024-06-25 22:53:32.949018243 +0200
+++ vendor/horstoeko/zugferd/src/ZugferdDocumentPdfBuilderAbstract.php	2024-06-25 22:53:49.064766548 +0200
@@ -200,7 +200,7 @@
         $pageCount = $this->pdfWriter->setSourceFile($pdfDataRef);
 
         for ($pageNumber = 1; $pageNumber <= $pageCount; ++$pageNumber) {
-            $pageContent = $this->pdfWriter->importPage($pageNumber, '/MediaBox');
+            $pageContent = $this->pdfWriter->importPage($pageNumber, '/CropBox',true,true);
             $this->pdfWriter->AddPage();
             $this->pdfWriter->useTemplate($pageContent, 0, 0, null, null, true);
         }
